### PR TITLE
add recommendation to review template early

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,11 @@ Create an enhancement here once you:
 - have identified people who agree to work on and maintain the enhancement
   - many enhancements will take several releases to complete
 
+Although you should probably not start your new idea's journey by writing an 
+enhancement up front, it's worth perusing the enhancement template to understand the
```
Comment (wking) [inline README.md:72]:
```suggestion
enhancement up front, it's worth perusing [the enhancement template](guidelines/enhancement_template.md) to understand the 
```

**Comment by dhellmann** [comment]
/approve

I'll leave it for @wking to lgtm with the link update.

**Comment by bparees** [comment]
@wking pulled in your suggestion

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**